internal/platform/activepodresources/supportversion.go

```diff
@@ -0,0 +1,104 @@
+package activepodresources
```

Comment:
missing copyright boilerplate (`hack/boilerplate.go.txt`) which makes me realize we don't have a CI rule which ensure all (new) files have the boilerplate, so we should add such CI rule (obviously not related to this PR)